.github/workflows/release.yml

```diff
@@ -49,3 +49,15 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+
+      - name: Get Package Version
+        run: echo "WRANGLER_VERSION=$(npm view wrangler@beta version)" >> $GITHUB_ENV
```

Comment:
Should we not be getting this version from the local `package.json`?

Reply:
This happens after the changeset does a release so it is the latest "released" version to pass to Sentry

Reply:
Done.

Reply:
This gets triggered for every commit that lands.